libmaven/mzSample.h

```diff
@@ -2166,11 +2167,11 @@ class MzKitchenProcessor{
                                  shared_ptr<MzkitchenMetaboliteSearchParameters> params,
                                  bool debug=false);
 
-    // TODO
-//    static void assignBestMetaboliteToGroup(PeakGroup* g,
-//                                            vector<CompoundIon>& compound,
-//                                            shared_ptr<MzkitchenMetaboliteSearchParameters> params,
-//                                            bool debug=false);
+
+    static void assignBestMetaboliteToGroup(PeakGroup* g,
+                                            vector<CompoundIon>& compounds,
+                                            shared_ptr<MzkitchenMetaboliteSearchParameters> params,
+                                            bool debug=false);
 };
```

Comment:
This function is used both by the MAVEN GUI and the multithreaded CLI program peakdetector, which runs on clusterbuddy and is connected to the mzkitchen shiny app.